app/serializers/search_and_compare/course_serializer.rb

```diff
@@ -139,7 +139,7 @@ def description_sections
        },
        {
          Name: "about this training provider accrediting",
-         Text: object.accrediting_provider_description
+         Text: object.accrediting_provider_description.present? ? object.accrediting_provider_description : ''
```

Comment:
Also an option to avoid a ternary:

```suggestion
         Text: object.accrediting_provider_description.to_s
```